src/network/network_udp.cpp

```diff
@@ -28,6 +28,7 @@
 #include "../newgrf_text.h"
 #include "../strings_func.h"
 #include "table/strings.h"
+#include <atomic>
```

Comment:
Is this still needed here? If so, shouldn't it be included by the header that actually requires it?

Reply:
`std::atomic` is used a bit lower, so putting it in the header file would be a bit weird. And needed yes/no .. ugh, our include-stuff is a mess with no order to it what-so-ever. But on the argument, keep the PR as small as possible, I will remove it again.